library.properties

```diff
@@ -6,5 +6,6 @@ sentence=I2Cwrapper is a generic modular framework for Arduino I2C target device
 paragraph=Consists of an easily extensible firmware framework and a I2C controller library. Ready to use modules exist for stepper motors, servo motors, digital/analog pins, ESP32 touch buttons, and TM1638 boards.
 category=Device Control
 url=https://github.com/ftjuh/I2Cwrapper
-architectures=avr,esp32,esp8266
+architectures=avr,esp32,esp8266,samd
 includes=I2Cwrapper.h
+depends=FlashStorage
```

Comment:
This dependency is only for samd, so I feel it makes no sense to claim it as a general dependency.

Note that a lot of other "could be" dependencies exist, such as all the libraries used by the modules. I left these out on purpose, as I don't want to urge users to install e.g. the TM1638lite library if they are only interested in AccelSTepperI2C.

Reply:
I debated that. Removed as request.